sample-storage/connections.yaml

```diff
@@ -85,20 +85,20 @@
     password: w8wfy99DvEmgkBsE
     database: plotly
     port: 50000
-    host: 35.184.35.183
+    host: 35.193.19.132
```

Comment:
Can you add hostnames to `test.plotly.host` and use those instead of IP addresses?

Reply:
(I suggest setting a TTL of 60 seconds when you do so.)